src/main/java/core/basesyntax/services/impl/WriteFileServiceImpl.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.services.impl;
+
+import core.basesyntax.dao.StoreDao;
+import core.basesyntax.services.WriteFileService;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Map;
+
+public class WriteFileServiceImpl implements WriteFileService {
+    private static final String COMMA = ",";
+
+    @Override
+    public boolean writeToFile(StoreDao dao, String fileName) {
+        Map<String, Integer> resultStore = dao.getStorage();
+        try (FileWriter writer = new FileWriter(fileName)) {
+            writer.write("fruit,quantity");
+            for (Map.Entry<String, Integer> entry : resultStore.entrySet()) {
+                writer.write(System.lineSeparator() + entry.getKey() + COMMA + entry.getValue());
+            }
+        } catch (IOException e) {
+            System.out.println("Cant write the data to the file " + fileName);

```

Comment:
You should throw unchecked exception here

Reply:
not fixed. just printing exceptions and stack traces is a bad practice. Throw runtime exception